### PR TITLE
Update Charter draft for 2025

### PR DESCRIPTION
**Pull request by nigelmegitt**
Using outcomes of discussion in [TTWG call 2024-11-07](https://www.w3.org/2024/11/07-tt-minutes.html#fa9d).

* Increment dates by +2 years
* Adjust expected meeting schedule
* Adjust line reformatting for length
* Update references to point to latest versions (including the Process) and use lower case styling
* Adjust examples paragraph for new normative specifications section on TTML2 profiles
* Add IMSC1.3 and DAPT as deliverables
* Add ARIB as an external organisation

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/charter-timed-text/pull/89.html" title="Last updated on Dec 11, 2024, 3:57 PM UTC (7015b33)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/charter-timed-text/89/f36ef73...7015b33.html" title="Last updated on Dec 11, 2024, 3:57 PM UTC (7015b33)">Diff</a>

### CODE REVIEW

**Comment by nigelmegitt** [comment]
@himorin please compare to latest Charter Template to check if we are missing any important deltas compared to the previous Charter, as discussed.

File: index.html [hunk 1]
```diff
@@ -91,15 +91,15 @@ <h1 id="title">Timed Text Working Group Charter</h1>
               Start date
             </th>
             <td>
-              8 April 2023
+              8 April 2025
```
Comment (himorin) [inline index.html]:
```suggestion
              <i class="todo">TBD</i>
```

File: index.html [hunk 2]
```diff
@@ -91,15 +91,15 @@ <h1 id="title">Timed Text Working Group Charter</h1>
...
             </td>
           </tr>
           <tr id="CharterEnd">
             <th>
               End date
             </th>
             <td>
-              7 April 2025
+              7 April 2027
```
Comment (himorin) [inline index.html]:
```suggestion
              <i class="todo">TBD (Start date + 2 years)</i>
```

File: index.html [hunk 3]
```diff
@@ -200,17 +202,39 @@ <h3>New Normative Specifications Continuously Under Development</h3>
...
-      <p>This specification specifies an Hypothetical Render Model (HRM) that constrains 
-        the complexity of an <a href="https://www.w3.org/TR/imsc-hrm/#dfn-imsc-document-instance">IMSC Document Instance</a>.</p>
-      <p class="draft-status"><b>Draft state:</b> <a href="https://www.w3.org/TR/imsc-hrm/">Working Draft</a></p>
+      <p>This specification defines two profiles of <a href="https://www.w3.org/TR/ttml2/">[ttml2]</a>:
+        a text-only profile and an image-only profile.
+        These profiles are intended to be used across subtitle and caption delivery applications worldwide,
+        thereby simplifying interoperability, consistent rendering and conversion to other subtitling and captioning formats.</p>
+      <p>This specification improves on <a href="https://www.w3.org/TR/ttml-imsc1.2/">[ttml-imsc1.2]</a> by supporting additional contemporary practices,
+        while retaining compatibility with <a href="https://www.w3.org/TR/ttml-imsc1.2/">[ttml-imsc1.2]</a> documents.</p>
+      <p class="draft-status"><b>Draft state:</b> <a href="https://www.w3.org/TR/ttml-imsc1.3/">Working Draft</a></p>
```
Comment (himorin) [inline index.html]: draft state should be none?

File: index.html [hunk 4]
```diff
@@ -200,17 +202,39 @@ <h3>New Normative Specifications Continuously Under Development</h3>
...
       <p><b>Adopted Working Draft:</b> <a href="https://www.w3.org/TR/2022/WD-imsc-hrm-20220106/">IMSC Hypothetical Render Model</a> (6 January 2022)</p>
```
Comment (himorin) [inline index.html]: there is no adopted working draft, afaiu. also remove one line below?

**Comment by himorin** [comment]
sorry for taking time to reach this issue...
@nigelmegitt can we have some (short) lines for [motivation and background](https://w3c.github.io/charter-drafts/charter-template.html#background) section, like [WebGPU one](https://gpuweb.github.io/admin/wg-charter.html)?
something from 2nd paragraph of scope could be enougth??

added per line comments as review, also

- IMSC 1.3 is new and to be moved into 2.1?
- DAPT exclusion draft as [FPWD](https://www.w3.org/TR/2023/WD-dapt-20230425/) and [annoucement](https://lists.w3.org/Archives/Public/public-tt/2023Apr/0010.html)
- switch HTML WG to WHATWG?
- following points will be updated during aligning to latest template (reminder for myself)
  - all process links to non-dated link https://www.w3.org/policies/process/
  - remove performance HR
  - update section numbers of Process (in decision policy and about)
  - add new row in charter history table

File: index.html (hunk 3 above)
Comment (nigelmegitt) [reply index.html]: Yes, right now it is none, but I hope that we create a working draft before we submit this charter draft for review.

File: index.html (hunk 4 above)
Comment (nigelmegitt) [reply index.html]: done in 7015b33 if I understood your comment correctly @himorin

File: index.html (hunk 2 above)
Comment (nigelmegitt) [reply index.html]: Done in 7015b33

File: index.html (hunk 1 above)
Comment (nigelmegitt) [reply index.html]: Done in 7015b33

**Review by nigelmegitt — COMMENTED** [review]
Thanks @himorin I think I've applied those changes

**Comment by nigelmegitt** [comment]
> sorry for taking time to reach this issue... @nigelmegitt can we have some (short) lines for [motivation and background](https://w3c.github.io/charter-drafts/charter-template.html#background) section, like [WebGPU one](https://gpuweb.github.io/admin/wg-charter.html)? something from 2nd paragraph of scope could be enougth??

Done in 7015b33

> added per line comments as review, also
> 
>     * IMSC 1.3 is new and to be moved into 2.1?

Done in 7015b33

>     * DAPT exclusion draft as [FPWD](https://www.w3.org/TR/2023/WD-dapt-20230425/) and [annoucement](https://lists.w3.org/Archives/Public/public-tt/2023Apr/0010.html)

Done in 7015b33

>     * switch HTML WG to WHATWG?

Not done - this is in a section headed "W3C Groups" - one option would be to remove it from there and add WHATWG into "External Organizations".

>     * following points will be updated during aligning to latest template (reminder for myself)
>       
>       * all process links to non-dated link https://www.w3.org/policies/process/
>       * remove performance HR
>       * update section numbers of Process (in decision policy and about)
>       * add new row in charter history table

Thanks, yes.

**Review by himorin — APPROVED** [review]

File: index.html (hunk 4 above)
Comment (himorin) [reply index.html]: Yeah, let me fix at the last moment of rechartering. (draft state may also need to be fixed, depends)

File: index.html (hunk 3 above)
Comment (himorin) [reply index.html]: sure!

File: index.html (hunk 2 above)
Comment (himorin) [reply index.html]: thx

**Comment by himorin** [comment]
For HTML WG, yes, you are right. I'll talk around and seek the best option to update (or keep).
Feel free to merge, (or to wait for this Thu meeting?). I'll take care of template-ish items, and also status updates on drafts, etc.

**Review by gkatsev — APPROVED** [review]

**Pull request merged**